src/NegativeInt.js

```diff
@@ -0,0 +1,37 @@
+import { GraphQLScalarType } from 'graphql';
+import { GraphQLError } from 'graphql/error';
+import { Kind } from 'graphql/language';
+
+function processValue(value) {
+  if (isNaN(value)) {
+    throw new TypeError(`Value is not a number: ${value}`);
+  }
+
+  if (!(value < 0)) {
+    throw new TypeError(`Value is not a negative number: ${value}`);
+  }
+
+  return parseInt(value, 10);
+}
+
+export default new GraphQLScalarType({
+  name: 'PositiveInt',
```

Comment:
I believe `name` is meant to be `NegativeInt`?

Reply:
Ditto.